app/api/datastore/v2/applications.rb

```diff
@@ -27,9 +27,28 @@ class Applications < Base
         desc 'Return applications with pagination.'
         params do
           use :pagination
+
+          optional(
+            :status,
+            type: String,
+            default: nil,
+            desc: 'The status of the application.',
+            values: CrimeApplication::STATUSES
+          )
+
+          optional(
+            :sort,
+            type: Symbol,
+            default: :desc,
+            desc: 'The status of the application.',
+            values: %i[desc asc]
```

Comment:
Is 'desc' in this file abbreviating both 'description' and 'descending'? - It is kinda clear on context, but not as clear as it could be.